### ISSUE TITLE
Style/FormatStringToken with parens

### ISSUE THREAD

**Issue by tejasbubane** [rubocop-hq/rubocop#7343]
`Style/FormatStringToken` does not behave well with parens.

eg.

This line flags rubocop offense:
```ruby
MSG = 'Prefer using `%{replacement}(%<arguments>s)` over `%{assertion}(true, %<arguments>s)`.'
```
```
lib/rubocop/cop/minitest/assert_truthy.rb:24:30: C: Style/FormatStringToken: Prefer annotated tokens (like %<foo>s) over template tokens (like %{foo}).
        MSG = 'Prefer using `%{replacement}(%<arguments>s)` over ' \
                             ^^^^^^^^^^^^^^
lib/rubocop/cop/minitest/assert_truthy.rb:25:17: C: Style/FormatStringToken: Prefer annotated tokens (like %<foo>s) over template tokens (like %{foo}).
              '`%{assertion}(true, %<arguments>s)`.'
                ^^^^^^^^^^^^
```

But replacing `{}` with `<>` raises invalid string exception:
```ruby
format(MSG, assertion: assertion, replacement: replacement(assertion), arguments: arguments)
``` 
```
ArgumentError: malformed format string - %(
```

--------

## Expected behavior

rubocop should not mark an offense

## Actual behavior

rubocop marks it as offense

## Steps to reproduce the problem

Refer: https://github.com/rubocop-hq/rubocop-minitest/pull/12

## RuboCop version

```
$ [bundle exec] rubocop -V
0.74.0 (using Parser 2.6.4.0, running on ruby 2.6.3 x86_64-darwin18)
```

**Comment by buehmann** [comment]
> But replacing `{}` with `<>` raises invalid string exception:

It looks like you forgot to add the type specifier `s`:

```ruby
MSG = 'Prefer using `%<replacement>s(%<arguments>s)` over `%<assertion>s(true, %<arguments>s)`.'
```

That is what the cop is suggesting.

**Comment by buehmann** [comment]
I think adding some quotation marks to the message of  `FormatStringToken` could help prevent this mistake.

**Comment by tejasbubane** [comment]
> MSG = 'Prefer using `%<replacement>s(%<arguments>s)` over `%<assertion>s(true, %<arguments>s)`.'

This works on `0.80.1`. Maybe I might have forgotten the type specifier `s`. Closing.
